src/main/java/edu/indiana/dlib/amppd/controller/AmpUserController.java

```diff
@@ -21,7 +22,7 @@
 	 */
   @CrossOrigin(origins = "*", allowedHeaders = "*")
   @RestController
-  @Slf4j 
+  @Log 
   public class AmpUserController{
```

Comment:
Not sure this change is intentional, but we are using slf4j instead of java util log. the details were explained in AMP-194. Can you revert this change (in other files too)? The API is very similar, so it should cause minimal changes in code.